### DISCUSSION TITLE
Grave BUG API v1

### DISCUSSION THREAD

**Discussion by domenicogioia** [fattureincloud/api#154] [Q&A] [answered]
### non funziona piu' /v1/acquisti/dettagli   
api.fattureincloud.it/v1/acquisti/dettagli   **non funziona piu'**

Siamo ancora utilizzatori della versione 1
Fino a pochi giorni fa con la API v1 siamo sempre stati in grado di recuperare il file XML per importarli nel ns sistema.
Adesso quando andiamo a recuperare il file xml usando il link indicato nella risposta di /v1/acquisti/dettagli riceviamo accesso negato. Ha funzionato per anni, per favore ripristinare urgentemente

**Comment by valmoz** [comment, marked as answer]
Buonasera,
questa è solo la community in cui si discute come effettuare l'integrazione delle API v2. Non trattiamo delle V1 in questo canale.

Per la segnalazione di problemi è sempre consigliabile contattare il Supporto Clienti, specialmente per quanto riguarda le v1 (che qui sono totalmente off topic); le consiglio quindi caldamente di scrivere al supporto clienti, allegando il link a questa discussione.

Le chiederei inoltre di aggiungere al thread con il supporto l'errore che viene restituito dalle API.

Per la risoluzione del problema comunque continuiamo a consigliare fortemente il passaggio alle V2, vista anche la [dismissione sempre più vicina](https://developers.fattureincloud.it/blog/v1-deprecation). Se possibile le chiederei quindi di dirci perché non avete ancora effettuato il passaggio, e se state riscontrando dei problemi nell'effettuare la migrazione.

Qui le informazioni per contattarci: [Support & Feature Request](https://developers.fattureincloud.it/docs/support#-api-bugs-problems-and-user-specific-questions)

Grazie
Mauro
